smart_proxy_dynflow.gemspec

```diff
@@ -21,7 +21,14 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '~> 2.5'
 
-  gem.add_runtime_dependency "logging"
+  gem.add_runtime_dependency('dynflow', "~> 1.1")
+  gem.add_runtime_dependency('logging')
+  gem.add_runtime_dependency('rack')
+  gem.add_runtime_dependency('rest-client')
+  gem.add_runtime_dependency('sd_notify', '~> 0.1')
```

Comment:
This was only used when running smart proxy dynflow core as a standalone service, also it is brought in by smart proxy itself